### ISSUE TITLE
Update README to discuss library aspect of repo

### ISSUE THREAD

**Issue by nathanielc** [influxdata/flux#2]
Now that the repo is intended to be consumed as a library primarily and as a binary secondarily we should update the README.

Perhaps we should create some flux binary that consumes CSV data from STDIN or a file and outputs to a file or STDOUT. Then the examples of how to use and play with Flux can be updated to use that binary instead of the webserver (which does not exist in this repo now).

**Comment by desa** [comment]
I've been trying to use flux as a library in a side project, and it's a bit hard to follow all of the pieces. I'm happy to work on this document with a bit of guidance.
